lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -816,11 +817,30 @@ def print_redis_state(self):
         Print the per-IMSI state in Redis data store on AGW
         """
         magtivate_cmd = "source /home/vagrant/build/python/bin/activate"
-        state_cli_cmd = "state_cli.py keys IMSI*"
-        redis_state = self.exec_command_output(
-                magtivate_cmd + " && " + state_cli_cmd)
-        print("Redis state is [\n", redis_state, "]")
-
+        imsi_state_cmd = "state_cli.py keys IMSI*"
+        redis_imsi_keys = self.exec_command_output(
+            magtivate_cmd + " && " + imsi_state_cmd
+        )
+        keys_to_be_cleaned = []
+        for key in redis_imsi_keys.split('\n'):
+            if "directory" not in key:
```

Comment:
nit: I missed this on my previous review, we should add a comment on here just to explain this is for ignoring directoryd records